hash/msx2_flop.xml

```diff
@@ -22145,196 +22134,1591 @@ HOMEBREW
 		</part>
 	</software>
 
-	<software name="nvmg9608">
-		<description>NV Magazine 1996-08 (Japan)</description>
-		<year>1996</year>
-		<publisher>&lt;doujin&gt;</publisher>
-		<info name="developer" value="Syntax" />
+	<software name="nvh1">
+		<description>NV Hokkaido Vol. 1 (Japan)</description>
+		<year>1995</year>
+		<publisher>Syntax</publisher>
+		<info name="alt_title" value="NV北海道#1"/>
+		<info name="usage" value="Requires a Japanese system."/>
+		<part name="flop1" interface="floppy_3_5">
+			<dataarea name="flop" size="737280">
+				<rom name="nv hokkaido vol 1 - syntax.dsk" size="737280" crc="76dc41e2" sha1="a1d472a4872490c56db506c615879ae750da7356"/>
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="nvmg1">
+		<description>NV Magazine 1 (Japan)</description>
+		<year>1992</year>
+		<publisher>Syntax</publisher>
+		<info name="alt_title" value="NVマガジン1号"/>
+		<info name="usage" value="Requires a Japanese system."/>
+		<part name="flop1" interface="floppy_3_5">
+			<dataarea name="flop" size="737280">
+				<rom name="nv magazine 1 - syntax.dsk" size="737280" crc="7fe2b042" sha1="397ac3feed7da3c58145f362aea3ffbb001f9860"/>
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="nvmg2">
+		<description>NV Magazine 2 (Japan)</description>
+		<year>1992</year>
+		<publisher>Syntax</publisher>
+		<info name="alt_title" value="NVマガジン2号"/>
+		<info name="usage" value="Requires a Japanese system."/>
+		<part name="flop1" interface="floppy_3_5">
+			<dataarea name="flop" size="737280">
+				<rom name="nv magazine 2 - syntax.dsk" size="737280" crc="656259a0" sha1="2b707101e0afd063f17eb67ae6e5d54772a81890"/>
+			</dataarea>
+		</part>
+	</software>
```

Comment:
Please use “1-gou” and “2-gou” in the descriptions (or “1-gō” and “2-gō” if this software list uses that style).

Reply:
Fixed